backend/apid/graphql/schema/event_filter.graphql

```diff
@@ -0,0 +1,60 @@
+"""
+A Filter is a filter specification.
+"""
+type EventFilter implements Node, Namespaced, Resource {
+  "The globally unique identifier of the record"
+  id: ID!
+
+  "Namespace in which this record resides"
+  namespace: String!
+
+  "Name is the unique identifier for an event filter."
+  name: String!
+
+  "metadata contains name, namespace, labels and annotations of the record"
+  metadata: ObjectMeta!
+
+  "Command is the command to be executed."
+  action: EventFilterAction!
+
+  "Env is a list of environment variables to use with command execution"
+  expressions: [String!]
```

Comment:
I think it's safe to say this won't ever return `null`. 

`expressions: [String!]!`